extensions.json

```
@@ -669,8 +669,8 @@
     {
       "id": "jebbs.plantuml",
       "repository": "https://github.com/qjebbs/vscode-plantuml",
-      "version": "2.14.0",
-      "checkout": "v2.14.0",
+      "version": "2.14.2",
+      "checkout": "master",
```

`2.14.0` still seems to be the latest release.

```suggestion
      "version": "2.14.0",
      "checkout": "v2.14.0",
```

---

Stopped auto-upgrading: https://github.com/open-vsx/publish-extensions/commit/e159c75a6636af164bf6c4d795c0a22684b1d10f